types/hashmap/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for HashMap 2.0.3
+// Type definitions for HashMap 2.3.0
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
Interesting enough, I noticed that for this package there are **a lot** of lints that have been explicitly disabled. Do you want me to enable the `dt-header` lint (and make the change), or you think that it is better to re-enable all the lints and fix all the errors? I never contributed to DefinitelyTyped before, I totally don't know the linting customization policies of the project.

Reply:
Usually whatever list rules are set when the package was created tends to stick around, so when I'm reviewing new packages I usually tell people to not disable any lints. For this package you should enable `dt-header`, but it's up to you whether you want to re-enable the others. Ideally they would all be enabled, but it could be time consuming to fix.

Reply:
I decided to re-enable all the lints, mainly because it was a matter of easy and pretty effortless fixes. There is just a fix I made I want to highlight: I changed the link of a contributor because it was not a github username url, as required by `dt-header`. I checked that the user I found was the actual contributor, but I am not sure if this kind of change without the permission of the author could be against some kind of policy. I resolve the conversation, but fill free to make more suggestions for a better PR.